unyt/tests/test_array_functions.py

```diff
@@ -1227,16 +1227,37 @@ def test_isclose_error():
         np.isclose(x, y)
 
 
-@pytest.mark.parametrize(
-    "func",
-    [
-        np.linspace,
-        np.logspace,
-        np.geomspace,
-    ],
-)
-def test_xspace(func):
-    res = func(1 * cm, 11 * cm, 10)
+@pytest.mark.parametrize("retstep", (True, False))
+def test_linspace(retstep):
+    res = np.linspace(1 * cm, 11 * cm, 10, retstep=retstep)
+    if retstep:
+        res, step = res
+    assert type(res) is unyt_array
+    assert res.units == cm
+    if retstep:
+        assert type(step) is unyt_quantity
+        assert step.units == cm
+
+
+def test_logspace_with_units_raises():
+    with pytest.raises(TypeError, match="The first argument"):
```

Comment:
I don't know off hand what the error message should be saying. Could you expand the `match` argument a little bit so it's understandable on its own ?